test/built-ins/Temporal/Calendar/prototype/yearMonthFromFields/throw-type-error-from-RequireInternalSlot.js

```diff
@@ -0,0 +1,17 @@
+// Copyright (C) 2021 the V8 project authors. All rights reserved.
```

Comment:
This test errors out for me.

Anyway, the brand check is covered more comprehensively by https://github.com/tc39/proposal-temporal/blob/main/polyfill/test/Calendar/prototype/yearMonthFromFields/branding.js, consider moving it over instead of this one.